about.html

```diff
@@ -0,0 +1,98 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Tesfaye-capstone-project</title>
+    <link href="https://fonts.googleapis.com/css2?family=Lato&family=Poppins&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css" />
+
+</head>
+<body>
+    <div class="header-container">
+        <div class="header-nav">
+                <ul>
+                <li><a href="#"><img src="images/icons/facebook.png" /> </a></li> 
+                <li><a href="#"><img src="images/icons/twitter.png" /> </a></li>
+                <li><a href="#">English </a></li>
+                <li><a href="#">My Page </a></li>
+                <li><a href="#">Logout </a></li>
+                </ul>
+        </div>
+    </div> 
+    <header>
+    <div class="nav-container">
+          <nav>
+            <a href="./index.html"><img src="images/CC_Global_Summit_2015-logo.png" /> </a>
+            <ul class="nav-links">
+                <li class="link-item"><a href="./index.html">Home</a></li>
+                <li class="link-item"><a href="#">Program</a> </li>
+                <li class="link-item"><a href="#">Join</a> </li>
+                <li class="link-item"><a href="#">Sponsor</a> </li>
+                <li class="link-item"><a href="#">News</a> </li>
+            </ul>
```

Comment:
- [ ] Your App is not responsive on the Mobile version. Kindly make it responsive. 

![image](https://user-images.githubusercontent.com/58504655/219674729-7adb78ab-7273-49e0-9645-82096530d66b.png)